lib/openstudio-standards/btap/structure.rb

```diff
@@ -555,80 +559,42 @@ def initialize(model = nil, activity = nil, massive = true)
         mass.setSpace(space)
       end
 
-      # Embodied CO2-e kg (A1-A3) of a model is tallied separately as follows:
-      @co2              = {}
-      @co2[:structure ] = 0
-      @co2[:insulation] = 0
-      @co2[:cladding  ] = 0
-
-      # The :structure key/value pair includes above grade 'structures' (e.g.
-      # slabs, columns) and 'framing' (e.g. wood-framed vs steel-framed). Why
-      # grouped together? In many smaller-scale facilities, structure and
-      # framing are synonymous, or at least tightly coupled, e.g.:
-      #   - load-bearing wood-framed walls in small-scale residential
-      #   - load-bearing CMU walls in small-scale industrial
-      #   - metal buildings
+      # Embodied CO2-e kg (A1-A3?) of a model's structure includes:
+      #   - non-modelled above grade items (e.g. columns)
+      #   - non-modelled partitions
       #
       # Below-grade structures (rebar + poured concrete) are ignored - no
       # alternative options are considered for the moment, e.g. lower carbon
       # concrete mixes.
-      #
-      # Upon initialization, only the :structure tally is set. It is assumed
-      # that structure (and main framing) do not change with NECB U-factor
-      # requirements, e.g.:
```

Comment:
Embodied carbon linked to modelled surfaces (e.g. envelope), now calculated elsewhere in BTAP.